.github/workflows/test.yml

```diff
@@ -9,7 +9,6 @@ on:
   pull_request:
 env:
   GO_VERSION: '1.16'
-  GO_VERSION_LEGACY: '1.14'
```

Comment:
do we want to add 1.17 as well?

Reply:
I assume that go 1.17 will be released in the next few weeks. Therefore I think we can switch from 1.16 directly to 1.17. Do you have thoughts on that?

Reply:
yeah that sounds good to me